pdchaosazure/machine/actions.py

```diff
@@ -161,19 +166,20 @@ def stress_cpu(filter: str = None,
     futures = []
     with concurrent.futures.ThreadPoolExecutor(max_workers=len(machines)) as executor:
         for machine in machines:
-            command_id, script_content = command.prepare(machine, 'cpu_stress_test')
+            command_id, script_content = command.prepare(machine, operation_name)
             parameters = {
                 'command_id': command_id,
                 'script': [script_content],
                 'parameters': [
-                    {'name': "duration", 'value': duration}
+                    {'name': "input_duration", 'value': duration},
```

Comment:
any reason for prefixing it with "input_"?

Reply:
Yes. Readability inside the script. From the perspective of the script it's more easy to see which values are *input*ed and which variables are calculated inside the script.